include/sKeyboardEvent.h

```diff
@@ -0,0 +1,37 @@
+#ifndef D2TM_SKEYBOARDEVENT_H
+#define D2TM_SKEYBOARDEVENT_H
+
+#include "allegroh.h"
+
+#include <string>
+#include "../utils/cPoint.h"
```

Comment:
```suggestion
#include "utils/cPoint.h"
```
CMake will take care of the path. And this file can be moved around to a different directory depth and things will still build.

Reply:
thx!